auth/src/main/java/org/aerogear/mobile/security/SecurityCheckType.java

```diff
@@ -72,9 +72,9 @@ public SecurityCheck getSecurityCheck() {
     /**
      * Gets the name of the security check.
      *
-     * @return {@link String} {@link SecurityCheck#getName()}
+     * @return {@link String} {@link SecurityCheck#getType()}
      */
     public String getName() {
```

Comment:
This seems confusing to me. It's a method called getName() that internally calls getType()